src/components/SelectableCard/index.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from '@emotion/styled'
+import React, { ChangeEventHandler, ReactNode } from 'react'
+import Checkbox from '../Checkbox'
+import Radio from '../Radio'
+
+const StyledElement = (element: typeof Radio) => styled(element, {
+  shouldForwardProp: prop => !['showTick'].includes(prop),
+})<{ showTick?: boolean }>`
+  display: inline-flex;
+  align-items: start;
+  padding: ${({ theme }) => theme.space['2']};
+  border-radius: ${({ theme }) => theme.radii.default};
+  border: 1px solid
+    ${({ theme, checked, disabled }) =>
+      checked && !disabled
+        ? theme.colors.primary.borderWeak
+        : theme.colors.neutral.borderWeak};
+  color: ${({ theme, checked }) =>
+    checked ? theme.colors.primary.textWeak : theme.colors.neutral.text};
+  ${({ theme, disabled }) =>
+    disabled ? `background: ${theme.colors.neutral.backgroundDisabled};` : null}
+
+  &:hover,
+  &:focus-within,
+  &:active {
+    border: 1px solid
+      ${({ theme, disabled }) =>
+        disabled
+          ? theme.colors.neutral.borderWeakDisabled
+          : theme.colors.primary.borderWeakHover};
+  }
+
+  svg {
+    ${({ showTick }) => (!showTick ? `display: none;` : null)}
+  }
+`
+
+type SelectableCardProps = {
+  name?: string
+  children:
+    | (({
+        disabled,
+        checked,
+      }: Pick<SelectableCardProps, 'checked' | 'disabled'>) => ReactNode)
+    | ReactNode
+  value: string | number
+  onChange: ChangeEventHandler<HTMLInputElement>
+  showTick?: boolean
+  type?: 'radio' | 'checkbox'
+  disabled?: boolean
+  checked?: boolean
+  className?: string
+}
+
+const SelectableCard = ({
```

Comment:
you should add isError or error props to be able to show visual error on this component and children when we will use it on a FormField

Reply:
@philibea I will add a isError for the error style, for an error message a SelectableCardGroup will be required